index.html

```diff
@@ -0,0 +1,244 @@
+
+                    </div>
+                    <ul class="menu-ul">
+                        <li class="menu-li">
+                            <a href="./about.html">About</a>
+                        </li>
+                        <li class="menu-li">Tickets</li>
+                        <li class="menu-li">Join</li>
+                        <li class="menu-li">Sponsor</li>
+                        <li class="menu-li">News</li>
+                        <li class="menu-li cc">CC Campaign</li>
+
+                    </ul>
+                </div>
+
+            </nav>
+        </div>
+
+    </header>
+    <main>
+        <section>
+            <div class="welcome">
+                <div class="intro">
+                    <h3> "Hello! Sharing World"</h3>
+                    <h1> "CREATIVE COMMONS <br> GLOBAL SUMMIT <br> 2015"</h1>
+                    <p>Praesent porttitor, mi quis pulvinar rutrum, nunc dolor rutrum enim, eu tempor nulla felis eu
+                        orci.
+                        Ut eu turpis nisi. Pellentesque convallis, purus vitae mollis ultrices, ex lectus consectetur
+                        nulla,
+                        et blandit ipsum nibh sit amet ipsum.</p>
+                    <h2> 2015.10.15(THU) ~ 16(FRI) <BR> @ National Museum of Korea, Art Center Nabi and More </h2>
+
+
+
+                </div>
+            </div>
+        </section>
+    </main>
+
+
+    <section id="about">
+        <div class="lets-grow">
+            <h3 class="lets-grow-title">Main Program</h3>
+            <ul class="programs">
+                <li class="Lecture">
+                    <img src="./img/Tools.png" alt="Tools">
+                    <h4>Lecture</h4>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="cc-Exhibition">
+                    <img src="./img/Bulb.png" alt="Bulb">
+                    <h4>CC Exhibtion</h4>
+                    <p>Lorem ipsum dolor sit amet.</p>
+
+                </li>
+                <li class="forum">
+                    <img src="./img/Handshake.png" alt="Forum">
+                    <h4>Forum</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="workshop">
+                    <img src="./img/Medal.png" alt="Medal">
+                    <h4>Workshop</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+
+                </li>
+                <li class="ignite">
+                    <img src="./img/Handshake.png" alt="Handshake">
+                    <h4> CC Ignite</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, corporis!</p>
+
+                </li>
+
+
+            </ul>
+            <div class="see-program"> <a href="#">See our Full Program</a>
+            </div>
```

Comment:
Kudos 👍 for removing the red background, just like the previous reviewer mentioned, kindly restyle the link in order to match up with the design.
| Your Design |Figma Design |
| :---:   | :---: | 
| <img width="221" alt="Screenshot 2022-11-24 at 20 47 44" src="https://user-images.githubusercontent.com/73854999/203857609-0da9d33e-b0d5-4dec-ab76-53421956bda0.png">|<img width="358" alt="Screenshot 2022-11-24 at 20 48 48" src="https://user-images.githubusercontent.com/73854999/203857639-5210b264-3853-44ea-90ad-ecbbf53a72d6.png">|